specification/ai/DocumentIntelligence/models.tsp

```diff
@@ -482,10 +472,13 @@ model AnalyzeResultOperation {
   analyzeResult?: AnalyzeResult;
 }
 
-@added(Versions.v2024_07_31_preview)
 @doc("Status and result of the analyze batch operation.")
 model AnalyzeBatchResultOperation {
-  @doc("Operation status.  notStarted, running, completed, or failed")
+  @doc("Analyze batch operation result ID.")
+  @path
+  resultId?: uuid;
+
+  @doc("Operation status.  notStarted, running, succeeded, or failed")
```

Comment:
Is this doc still accurate? `OperationStatus` seems having more values than mentioned here. Especially, we should call out the terminal values, if any of them is uncommon.

Reply:
This is accurate.  Skipper is only applicable to AnalyzeBatchOperationDetail.status, where it is explicitly documented.